### ISSUE TITLE
Compilation error with version 3.1.0

### ISSUE THREAD

**Issue by apupier** [lquixada/cross-fetch#94]
upgrading from 3.0.6 to 3.1.0, I'm hitting these compilation errors:

```
node_modules/cross-fetch/index.d.ts:18:14 - error TS2304: Cannot find name 'BodyInit'.

18   new(body?: BodyInit | null, init?: ResponseInit): Response;
                ~~~~~~~~

node_modules/cross-fetch/index.d.ts:18:38 - error TS2304: Cannot find name 'ResponseInit'.

18   new(body?: BodyInit | null, init?: ResponseInit): Response;
                                        ~~~~~~~~~~~~

node_modules/cross-fetch/index.d.ts:25:14 - error TS2304: Cannot find name 'HeadersInit'.

25   new(init?: HeadersInit): Headers;
```

environment versions:
```
(base) [apupier@fedora vscode-camelk]$ nvm version
v11.15.0
(base) [apupier@fedora vscode-camelk]$ npm --version
6.7.0
```

**Comment by lquixada** [comment]
@apupier thanks for reporting the issue! can you provide more info on how to reproduce that error? the typescript version would also be helpful.

**Comment by apupier** [comment]
* clone https://github.com/camel-tooling/vscode-camelk/ and pick branch dependabot/npm_and_yarn/cross-fetch-3.1.0
* npm install
* npm run compile

**Comment by apupier** [comment]
tsconfig.json:
```
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es6",
        "outDir": "out",
        "lib": [
            "es6"
        ],
        "sourceMap": true,
        "resolveJsonModule": true,
        /* Strict Type-Checking Option */
        "strict": true,   /* enable all strict type-checking options */
        /* Additional Checks */
        "noUnusedLocals": true /* Report errors on unused locals. */
        // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
        // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
        // "noUnusedParameters": true,  /* Report errors on unused parameters. */
    },
    "include": [
        "src"
    ],
    "exclude": [
        "node_modules",
        ".vscode-test",
        "test-resources"
    ]
}
```

**Comment by apupier** [comment]
typescript version ^4.2.3

https://github.com/camel-tooling/vscode-camelk/blob/bcc15a025af89d94ccea0277cddc5d3a817d32c2/package.json#L312

**Comment by pitgrap** [comment]
Same here. If I open the index.d.ts all 3 lines are marked "red" in my IDE.
`BodyInit` and `HeadersInit` are missing in the import, but `ResponseInit` is not existing at all. :(

Caused by PR #93

**Comment by lquixada** [comment]
ok, this is supposed to be fixed on 3.1.1. please let me know if other issues come up. closing this for now.
